packages/visualizations/src/components/Table/TableCard.svelte

```diff
@@ -20,10 +21,13 @@
         source,
         unstyled,
         locale: localeOption,
+        direction: directionOption,
         pagination,
     } = options);
     $: $locale = localeOption || navigator.language;
     $: defaultLoadingRowsNumber = pagination ? pagination.recordsPerPage : 5;
+    $: $direction = directionOption || (RTL_LANGUAGES_CODE.includes($locale) ? 'rtl' : 'ltr');
+    $: $isRtl = $direction === 'rtl';
     $: loadingRowsNumber = isLoading ? defaultLoadingRowsNumber : null;
```

Comment:
You could do with only one store here, $dir is never used again
```suggestion
    $: {
        const dir = direction || (RTL_LANGUAGES_CODE.includes($locale) ? 'rtl' : 'ltr');
        $isRtl = dir === 'rtl';
    }
```